twilio-flex-webchat/src/api.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * These endpoints are included client-side for demo purposes only.
+ * Do *not* include these endpoints client-side in production code to ensure API keys are not exposed.
```

Comment:
Maybe useful to link to our old instant server Glitch post, but up to you if we want to include URLs in code comments (might get outdated, so maybe not worth it?): https://www.daily.co/blog/deploy-a-daily-co-backend-node-js-server-instantly/